### PR TITLE
ebill_paynet: Add http timeout

### PR DESCRIPTION
**Pull request by TDu**

### CODE REVIEW

File: ebill_paynet/models/paynet_service.py [hunk 1]
```diff
@@ -53,6 +53,10 @@ class PaynetService(models.Model):
         readonly=True,
     )
     active = fields.Boolean(default=True)
+    operation_timeout = fields.Integer(
+        string="HTTP Timeout",
+        help="Timeout for each HTTP (GET, POST) request in seconds."
```
Comment (sebalix) [inline ebill_paynet/models/paynet_service.py]: Does it deserve a default value to not have a blocked Odoo worker indefinitely?

File: ebill_paynet/models/paynet_service.py [hunk 2]
```diff
@@ -62,13 +66,20 @@ def _compute_url(self):
             else:
                 record.url = SYSTEM_PROD_URL
 
+    def _get_dws_connection(self):
+        return PayNetDWS(
+            self.url,
+            self.use_test_service,
+            self.operation_timeout or None
```
Comment (mmequignon) [inline ebill_paynet/models/paynet_service.py]:
```suggestion
            operation_timeout=self.operation_timeout or None
```
This is a named argument.

**Review by mmequignon — APPROVED** [review]

File: ebill_paynet/models/paynet_service.py (hunk 1 above)
Comment (TDu) [reply ebill_paynet/models/paynet_service.py]: Maybe but how much is safe ?

Comment (TDu) [reply ebill_paynet/models/paynet_service.py]: Ok, I have set one

**Review by sebalix — APPROVED** [review]

File: ebill_paynet/models/paynet_service.py (hunk 2 above)
Comment (sebalix) [reply ebill_paynet/models/paynet_service.py]: This code already works like this too. What makes a keyword/named argument is the usage, not the declaration of the function/method. All parameters here could be named arguments, or positional ones (like in this case).
```python
PayNetDWS(operation_timetout=10, test_service='test', url='test')    # named arguments
# same than
PayNetDWS('test', 'test', 10)    # positional arguments
```

Comment (mmequignon) [reply ebill_paynet/models/paynet_service.py]: Yes I know, that's why I approved the PR anyway.

**Review by yvaucher — APPROVED** [review]

**Pull request merged**